api/src/main/java/org/openmrs/module/insuranceclaims/api/model/InsuranceClaim.java

```diff
@@ -93,6 +93,10 @@
 	@Column(name = "claim_status", nullable = false)
 	private InsuranceClaimStatus claimStatus;
 
+	@ManyToOne
+	@JoinColumn(name = "bill")
+	private Bill bill;
```

Comment:
There probably need to be added constraint nullable=false but this need to be discussed with rest of the team

Reply:
constraint added